docs/changelog/1_0_b1.rst

```diff
@@ -103,7 +103,7 @@ schema to be:
     };
 
 And we apply the changes by using :ref:`edgedb
-create-migration <ref_cli_edgedb_create_migration>` and :ref:`edgedb
+create-migration <ref_cli_edgedb_migration_create>` and :ref:`edgedb
```

Comment:
You've only changed reference but not the actual command

Reply:
This is deliberate, because this is a changelog reference to the command as it used to be called when it was introduced, but linked to the actual non-deprecated version of the command.

Reply:
Oops. Sorry, you're right!